disttools/librflxlang.gpr

```diff
@@ -0,0 +1,159 @@
+
+
+
+
+
+with "gnatcoll";
+with "gnatcoll_gmp";
+with "gnatcoll_iconv";
+
+
+
+
+
+
+library project Librflxlang is
+
+   for Create_Missing_Dirs use "true";
+   type Build_Mode_Type is ("dev", "prod");
+   Build_Mode : Build_Mode_Type := external ("BUILD_MODE", "dev");
+
+   type Library_Kind_Type is ("static", "relocatable", "static-pic");
+   Library_Kind_Param : Library_Kind_Type := external
+     ("LIBRARY_TYPE", external ("LIBADALANG_LIBRARY_TYPE", "static"));
```

Comment:
This file looks generated, but seems not to be generated anymore. How does this project file need to be handled in case of langkit updates? Should that be documented?

Reply:
Yes, good point. I explained the steps to recreate it in 7c26eea396c5623302c14242a9ab8dfbfdfe78c8.